src/rpc/blockchain.cpp

```diff
@@ -2012,6 +2032,12 @@ UniValue scantxoutset(const JSONRPCRequest& request)
             "          }\n"
             "        },\n"
             "      ]\n"
+            "3. \"options\"                               (object, optional)\n"
+            "      \"rawsweep\": {                        (object, optional) Optionally creates a raw sweep transaction\n"
```

Comment:
In commit "scantxoutset: Add optional raw sweep transaction".

Is this the best place to implement this? As a separate wallet RPC it could work much more correctly (as it would know the actual scripts involved and be able to give a vsize estimate that's more than a dumb guess), and be more useful too by being combinable with whatever way to find inputs.

I feel this makes the RPC look like a kitchen sink.

Reply:
Agree. Will remove it and PR it later as a separate method

Reply:
From https://github.com/bitcoin/bitcoin/pull/12196#issuecomment-393057791

> > tx = scantxoutset("start", { "sweep_to": address })["sweep_tx"]
> > fundrawtransaction(tx, { "subtractFeeFromOutputs": [0] })
>
>  @promag: would that also work without the wallet? My idea was to make `scantxoutset` work without a wallet.

@sipa not sure if you are suggesting something similar to:
```
unspents = scantxoutset("start", ...)
tx = createrawtransaction unspents ...
tx = fundrawtransaction ...
```
And how could it work without the wallet?

Reply:
@promag, I just mean scantxoutset + createrawtransaction. Then a separate wallet RPC like "walletestimatesignedrawtransactionsize" (shorter names welcome...) can be used to guess the size of the tx ahead of time. "fundrawtransaction" already does this estimation internally.

> Would that also work without the wallet?

No, that's the point. You cannot implement that without knowledge of the scripts involved.

It may be better to integrate this into PSBT, as it means there doesn't need to a single wallet that knows about all scripts in a transaction.